content/en/docs/contribute/style/style-guide.md

```diff
@@ -38,6 +38,32 @@ The English-language documentation uses U.S. English spelling and grammar.
 
 {{< comment >}}[If you're localizing this page, you can omit the point about US English.]{{< /comment >}}
 
+## Guidelines for Contributing content
+
```

Comment:
hello @aimeeu . A few style nits:
- What does it mean to `host` content from projects ... ? Does this mean that CNCF incubating projects can publish their docs on kubernetes.io? 
- Check the style guide about capitalization and headings, `Guidelines for contributing content`
- Use of slashes: stale/rotten
- Is it worthwhile to include the detailed info about archived projects? Possibly just state that content, for CNCF archived projects, is not supported
- Html vs Markdown table?

Reply:
@kbhawkey

-  I used the verbiage from the [Issue](https://github.com/kubernetes/website/issues/15576); I'll ask Zach.
-  Thank you for catching the capitalization typo
- Again, I used the verbiage from the Issue; maybe I just follow requirements too closely...lol
- I think it is worthwhile to include the info about how to figure out if a project has been archived since the CNCF doesn't maintain a nice list. The info is also useful to reviewers and approvers who may need to check project status. In the case of rkt, nothing on the website or in the GitHub repo stated the project was going to be archived.
- HTML vs Markdown -> the Style Guide is the only place that uses HTML tables. This was discussed in the previous [PR](https://github.com/kubernetes/website/pull/15892) for this issue (the PR that I had to close due to unfixable git squash problems).  I plan to open an issue to change the HTML tables to Markdown after this PR has been merged.

Reply:
@kbhawkey 👋 
> Does this mean that CNCF incubating projects can publish their docs on kubernetes.io?

No. 

Thanks for all the style guide feedback!

Reply:
@aimeeu 
> I used the verbiage from the Issue

That's a reasonable place to start, but as @kbhawkey and @sftim correctly review, the style guide takes precedence. My own notes are a shorthand at best, and should not be regarded as fit for human consumption. 😜

Reply:
>     * What does it mean to `host` content from projects ... ? Does this mean that CNCF incubating projects can publish their docs on kubernetes.io?

@kbhawkey raises an excellent question here. I think we should be as clear as possible. I would suggest using the word "include" instead of "host" (see explanation and suggested wording below).

In addition, we should explicitly call out the difference between "hosting" third-party content (the technical verbiage is actually included in the kubernetes/website source repository and published to docs.k8s.io) and linking to third-party content.

Perhaps a paragraph like this would work?

===

The Kubernetes documentation comprises the content of the [kubernetes/website](github.com/kubernetes/website) 
source repository. The majority of the Kubernetes documentation is specific to the Kubernetes project. However, there are some scenarios where the Kubernetes documentation includes content from or about non-Kubernetes projects.

This content falls into the following categories:

1) Instructional content involving non-Kubernetes projects during setup or operation of Kubernetes
2) Detailed technical content about how to use a non-Kubernetes project or how that project is designed
3) Content that describes a non-Kubernetes project
4) Content that simply links to information about a non-Kubernetes project

.... and then we go into specific guidance on handling the above categories of external content ...

thoughts?

Reply:
@jaypipes I reworked the entire section (lost myself for almost a day down the rabbit hole, at one point even started a flow diagram LOL). Please let me know if my changes are what you envisioned and clear for the average contributor.

Reply:
Do you think this content deserves a new page now, outside of the [style guide page](https://kubernetes.io/docs/contribute/style/style-guide/)? There is a lot of content now.

Reply:
It probably does. I'll wait for others to weigh in before making further changes. After a quick glance, a couple of other pages would need updating to reflect looking at both content guide and style guide before contributing.